mh/pat.py

```diff
@@ -1800,7 +1860,15 @@ def sendNtcFriendAccept(self, recipient_id, info, message, seq):
         info.unk_long_0x02 = pati.Long(20)
         data += info.pack()
         data += pati.lp2_string(message)
-        self.send_packet(PatID4.NtcFriendAccept, data, seq)
+
+        users = self.session.find_users(recipient_id, "", 1, 1)
+        if len(users) == 0:
+            self.sendAnsAlert(PatID4.NtcFriendAccept, "<LF=8><BODY><CENTER>Player is Offline.<END>", seq)
```

Comment:
Same deal here. Send alert before the transaction is finished.

Reply:
Addressed in latest commit